webportal/lambda/.chalice/deployed/test.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
I had created a test environment... but this file never got committed because it was .gitignored. I hand created this one, and chalice appears to have picked it up.